src/injected/client-view-controller.ts

```diff
@@ -27,10 +28,12 @@ export class ClientViewController {
     private visualizationStore: BaseStore<VisualizationStoreData>;
     private assessmentStore: BaseStore<AssessmentStoreData>;
     private tabStore: BaseStore<TabStoreData>;
+    private userConfigurationStore: BaseStore<UserConfigurationStoreData>;
     private scanResultStore: BaseStore<VisualizationScanResultData>;
     private currentScanResultState: VisualizationScanResultData;
     private currentAssessmentState: AssessmentStoreData;
     private currentTabState: TabStoreData;
+    private currentUserConfigStoreState: UserConfigurationStoreData;
```

Comment:
How is this being used? I didn't find meaningful usages of this, just assign the state from the store and check for nullity

Reply:
resolve as discussed offline